.circleci/README.md

```diff
@@ -0,0 +1,79 @@
+
+The config is split based on pipeline tasks. There are separate directories for `build-and-test`, `infrastructure`, `deployment`, `owasp`, and `util`. These are described below.
+
+### Directory structure
+
+#### build-and-test
+Contains workflows, jobs, and commands for building and testing the application.
+
+#### infrastructure
+Contains workflows, jobs, and commands for setting up the infrastructure on Cloud gov.
+
+#### deployment
+Contains workflows, jobs, and commands for deploying the application on Cloud gov.
+
+#### owasp
+Contains workflows, jobs, and commands for running OWASP scans on the application in during the pipelien and on Cloud gov environments.
+
+#### util
+Contains workflows that pertain to specific tasks, like `erd`, and jobs and comands that are used by multiple sections of the pipeline, like `cf-check`. This is important to allow different versions of the generated_config.yml to have access to all the needed jobs and commands.
+
+### Navigation examples
+What if you want to look at the jobs that we used to build and test the application?
+
+This is easy. We first go to the `build-and-test` directory then open the jobs.yml
+
+`build-and-test/ -> jobs.yml`
+
+<hr />
+
+What if you want to know how Terraform is used by our pipeline?
+
+Terraform is used to set up infrastruture so we would start in the `infrastructure` directory. To see how terraform is actualy used you would look in commands since that is generally where Circle Ci actually runs things. Within the `deploy-infrastructure` command in the commands.yml you can see that a terraform orb is beign used to run terraform commands.
+
+`infrastructure/ -> commands.yml -> deploy-infrastructure`
+
+<hr />
+
+What if you want to know when the how deployments are triggered?
+
+We know we want the `deployments` directory. To see how a pipeline is started we look at workflow.yml. Then we find the desired workflow.
+
+`deployment/ -> workflows.yml -> desired workflow`
+
+## Setting up scheduled pipeline for nightly OWASP scans
+We want a scheduled pipeline that runs once a day with the pipeline parameter `run_nightly_owasp_scan` set to `true`. You can use [these instructions](https://circleci.com/docs/scheduled-pipelines/#project-settings) from the docs or follow the instructions below.
+
+In Circle Ci go to `Project Settings -> Triggers -> Add Triggers`
+
+You want the Trigger Source to be scheduled.
+
+You want Repeats to be weekly.
+
+You want to select all for days and months.
+
+You want the start time to be at 0:00 UTC.
+
+You want to repeat once per hour.
+
+You want to set the branch to be the branch you want this scan to be run on.
```

Comment:
We only ever want this to be `develop`, no? Also can't the nightly scan be triggered against the deployed sites like prod and staging? Or is that a different workflow?

Reply:
The branch is required. We want to do it on the environments associated with Raft develop, HHS main, and HHS master. The Pipeline scheduler requires a branch although I am unsure how it is used. I think its just to associate the schedule with a branch.